scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -712,7 +713,9 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
                 <td>
                   <ul class="p-0" style="list-style-type: none;">
                   {% for disease_term in gene.disease_terms %}
-                    <li class="d-flex align-items-baseline"><span class="badge bg-secondary m-1">{{ disease_term._id}}</span>&nbsp;<span>{{ disease_term.description }}&nbsp;{{ disease_term.inheritance }}</span></li>
+                    <li class="d-flex align-items-baseline"><span class="badge bg-secondary m-1">{{ disease_term._id}}</span>&nbsp;<span>{{ disease_term.description }}&nbsp;{% if disease_term.source != 'ORPHA' and disease_term.inheritance %}
+                      {% for model in disease_term.inheritance %} {{ inheritance_badge(model,inherit_palette) }}{% endfor %}
```

Comment:
Wound't be confusing that variant page shows different values for in the inheritance?

Reply:
We have modified the variant page to accomodate this, adding a warning. Also, the more clear cut "Gene models" card is nearby. The variant page is used by very or at least somewhat experienced genetics and genomics persons, whereas the reports are often read also by doctors in training, who in particular provide medical input. In the not-yet-specialist case could be confused by an authoritative looking inheritance model, and cannot always be taken to understand the distinction between different database.
<img width="550" alt="Screenshot 2024-09-18 at 09 55 57" src="https://github.com/user-attachments/assets/286c96a4-a1c7-441f-bdb4-ca25430eda66">

Reply:
Yes I remember this problem and our solution.